kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/Altoida_converter/AltoidaAccelerometerConverter.kt

```diff
@@ -0,0 +1,23 @@
+package org.radarbase.connect.upload.converter
+
+import RADAR-Schemas.commons.connector.altoida.AltoidaAcceleration
+
+class AltoidaAccelerometerConverter(override val sourceType: String = "acceleration", val topic: String = "altoida-acceleration")
```

Comment:
Please make sure the topic names match those in the RADAR-Schemas specification.